### PR TITLE
Fix tooltip zindex

### PR DESCRIPTION
**Pull request by ojame**
tooltip

Half a fix for macropodhq/stack-client/issues/614

### CODE REVIEW

File: packages/tooltip/example.jsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ module.exports = React.createClass({
 
   render() {
     return (
-      <span data-tooltip="Tooltip!">Hover over me</span>
+      <div>
+        <span data-tooltip="Tooltip!">Hover over me (right tooltip)</span> <br />
+        <span data-tooltip-left="Tooltip!">Hover over me (left tooltip)</span>
+      </div>
```
Comment (grapegravity) [inline packages/tooltip/example.jsx]: Is it worth making the left variation an additional property? Seems like something which might be needed to be toggled? (i.e. `data-tooltop="Tooltip!" data-tooltip-left={true}`)

Comment (ojame) [inline packages/tooltip/example.jsx]: I'm not particularly sure, I don't have a case either way. @nelix?

Comment (nelix) [inline packages/tooltip/example.jsx]: I don't really care either way. I kinda hate the way this component works. So don't get me started :P

Comment (ojame) [inline packages/tooltip/example.jsx]: Interested to know how you'd prefer this component works?

Comment (grapegravity) [inline packages/tooltip/example.jsx]: “no icky CSS” is my guess :stuck_out_tongue_closed_eyes:

Comment (nelix) [inline packages/tooltip/example.jsx]: The component works, lets not get distracted with how it should work just yet

Comment (nelix) [inline packages/tooltip/example.jsx]: you got me @grapegravity :D

**Pull request merged**